code/game/objects/items/binoculars.dm

```diff
@@ -16,13 +16,13 @@
 	zoom(user, 11, 12)
 
 #define MODE_CAS 0
-#define MODE_RANGE_FINDER 1
-#define MODE_RAILGUN 2
-#define MODE_ORBITAL 3
+#define MODE_RAILGUN 1
+#define MODE_ORBITAL 2
+#define MODE_RANGE_FINDER 3
 
 /obj/item/binoculars/tactical
 	name = "tactical binoculars"
-	desc = "A pair of binoculars, with a laser targeting function. Alt+Click to toggle mode. Ctrl+Click when using to target something."
+	desc = "A pair of binoculars, with a laser targeting function. Alt+Click or space to toggle mode. Ctrl+Click when using to target something. Shift+Click to get coordinates. Ctrl+Shift+Click to fire OB when lasing in OB mode"
```

Comment:
"space" maybe should be "special action" if it is indeed referring to a bindable hotkey.

Reply:
Ah yes, mistake